CONTRIBUTING.md

```diff
@@ -146,6 +146,7 @@ You can download Eclipse and load the Frank!Framework sources into it using the
 - Press Install / Update.
 - If you have trouble with these instructions, then you can get help on the https://projectlombok.org/ site. On the top menu, choose "install" | "Eclipse".
 - The Frank!Framework can only run on Java 8. Please install a Java 8 JDK in addition to the JRE that is included in your Eclipse installation. You can find it [here](https://www.oracle.com/nl/java/technologies/javase/javase8u211-later-archive-downloads.html).
+- You may want to update m2e to prevent that Eclipse downloads unnecessary source jar files. Go to Help | Install New Software... . In the "Work with" field, enter the following link: https://download.eclipse.org/technology/m2e/releases/latest/. Then follow the steps.
```

Comment:
Dit is een bug in M2E en is opgelost in de nightly release build. Upgraden naar deze build lost het inderdaad op. Ik zie dit niet als een fix maar meer een tijdelijke fix die je kan doen totdat er een nieuwe versie van m2e uit komt.
Zie ook:
- https://bugs.eclipse.org/bugs/show_bug.cgi?id=576112
- https://github.com/eclipse-m2e/m2e-core/issues/331
- https://github.com/eclipse-m2e/m2e-core/issues/252

You may want to, is dus niet hoe deze zin moet beginnen, en dit is dan ook niet de plek waar deze tekst hoort te staan.
Verder hoe je m2e kan updaten, staat ook hier uitgelegd: https://github.com/eclipse-m2e/m2e-core#-installation